sites/public/src/components/shared/TermsModal.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from "react"
+import type { UseFormMethods } from "react-hook-form"
+import { Field, t } from "@bloom-housing/ui-components"
+import { Button, Dialog, Heading } from "@bloom-housing/ui-seeds"
+import Markdown from "markdown-to-jsx"
+import accountStyles from "../../../styles/create-account.module.scss"
+
+export type FormSignInValues = {
+  email: string
+  password: string
+}
+
+export type FormVerifyValues = {
+  code: string
+}
+
+export type TermsModalControl = {
+  errors: UseFormMethods["errors"]
+  handleSubmit: UseFormMethods["handleSubmit"]
+  register: UseFormMethods["register"]
+}
+
+export interface TermsModalProps {
+  control: TermsModalControl
+  onSubmit: (data: FormSignInValues | FormVerifyValues) => void
+  notChecked: boolean
+  setChecked: React.Dispatch<React.SetStateAction<boolean>>
+  openTermsModal: boolean
+  setOpenTermsModal: React.Dispatch<React.SetStateAction<boolean>>
+  isTermsLoading?: boolean
+}
+
+const TermsModal = ({
+  control: { errors, register, handleSubmit },
+  onSubmit,
+  notChecked,
+  setChecked,
+  openTermsModal,
+  setOpenTermsModal,
+  isTermsLoading,
+}: TermsModalProps) => {
+  return (
+    <>
+      <Dialog
```

Comment:
This should have some aria attributes with ids on the header & content for accessibility, if you check out other Dialogs there should be some examples!

Reply:
Odd there wasn't any in the create-account as this is just code moved from there. Will add in.